### ISSUE TITLE
output subtitles still out of sync

### ISSUE THREAD

**Issue by Crowley007** [smacke/ffsubsync#174]
[
[S01E07 - WEBDL-1080p NTb 5.1.mkv.2023-02-12-12-49-54.tar.gz](https://github.com/smacke/ffsubsync/files/10715921/S01E07.-.WEBDL-1080p.NTb.5.1.mkv.2023-02-12-12-49-54.tar.gz)
](url)

**Additional context**
Made with Bazarr. ffsubsync.log contains a lot of totally unrelated [Errno 2] No such file or directory: errors

**Comment by Crowley007** [comment]
I also tried with the standalone ffsybsync, as expected it fails here as well.

**Comment by smacke** [comment]
It looks like for this particular video, ffsubsync tries to fix the framerate -- it's possible the framerate is already fine though. Have you tried passing --no-fix-framerate to determine if that's the issue?

If there is an issue with framerate scaling, it's possible that the fixed scaling factors that ffsubsync tries by default didn't work. In this case, you can try passing `--gss`, which tries to use [golden section search](https://en.wikipedia.org/wiki/Golden-section_search) to fix the framerate.

Please let me know if either of those flags can help resolve your issue.

**Comment by Crowley007** [comment]
Yeah, the fps is wrong 24 vs 23.976, it should be fixed. But that is not the main issue, it is off by several second from the start when "fixed". My guess is ffsubsync mistakes the mtv logo jingle with voice.
